.github/workflows/acceptance-tests-runner.yml

```diff
@@ -96,7 +96,6 @@ on:
         required: true
     
 env:
-  TF_ACC: 1
```

Comment:
while redundant I do see the value in leaving the TF_ACC env var just for clarity

Reply:
don't know, if we assume "make testacc" does it i would remove it from here.

i would choose only one place to have it

Reply:
[q] Does this mean that TF_ACC is no longer needed to run acceptance tests? Including locally? If so, we should update Terraform Development Guide

Reply:
+1 I see the value in keeping it

Reply:
@oarbusi  testacc makefile goal is: TF_ACC=1 go test ...

Reply:
change reverted, i'll keep it, thx for the feedback